public/resources/ts/stats-defer.ts

```diff
@@ -79,18 +76,19 @@ if (calculated.skin_data) {
   const basicCapeRotationX = Math.PI * 0.06;
 
   if (!window.matchMedia("(prefers-reduced-motion: reduce)").matches) {
-    skinViewer.animations.add((player, time) => {
-      // Multiply by animation's natural speed
-      time *= 2;
-
-      // Arm swing
-      const armRotation = Math.cos(time) * 0.03 + basicArmRotationZ;
-      player.skin.leftArm.rotation.z = armRotation;
-      player.skin.rightArm.rotation.z = armRotation * -1;
-
-      // Cape wave
-      player.cape.rotation.x = Math.sin(time) * 0.01 + basicCapeRotationX;
-    });
+    // skinViewer.animation.add((player, time) => {
```

Comment:
also what's the point of this code? Either remove it or uncomment :shrug:

Reply:
can you just remove it then? there's no point in having it then

Reply:
sure